.travis.yml

```diff
@@ -1,12 +1,14 @@
 sudo: required
 
+language: ruby
```

Comment:
I'll try with C.

Reply:
I think it can be removed, as we are not actually building anything (now that it is C). Maybe if required, rather use

```yaml
language: python
```

as most of our static tests are based on it.

Reply:
The Travis image does not matter because we launch our own docker on top. I'll try the `minimal` image.